File: ios-cloudfiles-example/ios-cloudfiles-example/ViewController.m
//
//  ViewController.m
//  ios-cloudfiles-example
//
//  Created by Juan Carrera on 12/17/15.
//  Copyright © 2015 Juan Carrera. All rights reserved.
//

#import "ViewController.h"
#import <ios-cloudfiles/RSClient.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    RSClient *client = [[RSClient alloc] initWithProvider:RSProviderTypeRackspaceUS username:@"charlesofdl" apiKey:@"834cdac66be13c28109050b0704cc7a3"];
    
    [client getCDNContainers:^(NSArray *containers, NSError *jsonError) {
        
        NSLog(@"containers %@", containers);
        
    } failure:^(NSHTTPURLResponse * response, NSData *data, NSError *error) {
        
    }];
    

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: ios-cloudfiles-example-swift/ios-cloudfiles-example-swift/ViewController.swift
//
//  ViewController.swift
//  ios-cloudfiles-example-swift
//
//  Created by Juan Carrera on 12/17/15.
//  Copyright © 2015 Juan Carrera. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let client = RSClient(provider: RSProviderTypeRackspaceUK, username: "charlesofdl", apiKey: "834cdac66be13c28109050b0704cc7a3")
        
        client.getCDNContainers({ (containers, jsonError) in
            print(containers)
        }, failure: { (response, data, error) in
            print(error)
        })
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

